src/components/TodoItem.module.css

```diff
@@ -0,0 +1,23 @@
+.item {
+    font-size: 1.2rem;
+    list-style-type: none;
+    padding: 17px 0px;
```

Comment:
- [ ] please in CSS3, zero has no unit, can you please try to replace **0px** to **0** in all your CSS files to avoid the linter errors? :joy_cat: